test/distributed/_tensor/test_dtensor.py

```diff
@@ -542,6 +546,51 @@ def test_dtensor_save_load(self):
         reloaded_st = torch.load(buffer, weights_only=True)
         self.assertEqual(sharded_tensor, reloaded_st)
 
+    @with_comms
+    @unittest.skipIf(
+        IS_FBCODE,
+        "subprocess import torch fails with ModuleNotFoundError: No module named 'torch' in fbcode",
+    )
+    def test_dtensor_save_load_import(self):
+        for should_import in [True, False]:
+            device_mesh = self.build_device_mesh()
+            placements = [Shard(0)]
+            local_tensor = torch.randn(3, 3)
+            sharded_tensor = DTensor.from_local(local_tensor, device_mesh, placements)
+            with tempfile.NamedTemporaryFile() as f:
+                torch.save(sharded_tensor, f)
+                import_string = (
+                    "import torch.distributed.tensor" if should_import else ""
+                )
+                script = f"""
+import torch
+{import_string}
+x = torch.load("{f.name}", weights_only=True)
+"""
+                try:
+                    subprocess.check_output(
+                        [sys.executable, "-c", script],
+                        cwd=os.path.dirname(os.path.realpath(__file__)),
```

Comment:
Instead of specifying a working directory, why not path file via absolute path to the script?

Reply:
For this one actually added this following what was done here, copied over the comment as well

https://github.com/pytorch/pytorch/blob/175ba9fed6a718dd1d0add9c733ed49bab97cc6c/test/test_autograd.py#L10170-L10172